boilerplate/_lib/ensure.sh

```diff
@@ -22,6 +22,14 @@ golangci-lint)
         curl -sfL "${DOWNLOAD_URL}" | tar -C "${GOPATH}/bin" -zx --strip-components=1 "golangci-lint-${GOLANGCI_LINT_VERSION}-${GOOS}-amd64/golangci-lint"
     fi
     ;;
+venv)
+    # Set up a python virtual environment
+    python3 -m venv .venv
```

Comment:
We should be ensuring the python3 is installed explicitly and obtain the absolute path to it before attempting the venv install.

Reply:
Agree. That should be covered (along with installing the libs) as part of https://issues.redhat.com/browse/OSD-5074 / https://issues.redhat.com/browse/OSD-5096 / etc.